### PR TITLE
build(rust): Move cross config

### LINKED ISSUE #3626: Move `cross` configuration to Cargo.toml and `tools/cross`
We use [cross](https://github.com/cross-rs/cross) to cross compile ockam command binaries. 

Currently the configuration from cross lives in [Cross.toml](https://github.com/build-trust/ockam/blob/develop/Cross.toml), [Cross.Dockerfile.aarch64](https://github.com/build-trust/ockam/blob/develop/Cross.Dockerfile.aarch64) & [Cross.Dockerfile.armv7](https://github.com/build-trust/ockam/blob/develop/Cross.Dockerfile.armv7)

https://github.com/cross-rs/cross#configuration

1. We want to move the config in [Cross.toml](https://github.com/build-trust/ockam/blob/develop/Cross.toml) to [Cargo.toml](https://github.com/build-trust/ockam/blob/develop/Cargo.toml)
https://github.com/cross-rs/cross#option-1-configuring-cross-directly-in-your-cargotoml

2. We want to move [Cross.Dockerfile.aarch64](https://github.com/build-trust/ockam/blob/develop/Cross.Dockerfile.aarch64) & [Cross.Dockerfile.armv7](https://github.com/build-trust/ockam/blob/develop/Cross.Dockerfile.armv7) to `tools/cross` directory.
https://github.com/cross-rs/cross#dockerfiles

3. We want the FROM line in [Cross.Dockerfile.aarch64](https://github.com/build-trust/ockam/blob/develop/Cross.Dockerfile.aarch64) & [Cross.Dockerfile.armv7](https://github.com/build-trust/ockam/blob/develop/Cross.Dockerfile.armv7) to be pinned to the SHA of the base image

### PR DESCRIPTION
**Pull request by duttaANI**
## Current Behavior
Currently the configuration from cross lives in [Cross.toml](https://github.com/build-trust/ockam/blob/develop/Cross.toml)

## Proposed Changes
To move config in Cross.toml to Cargo.toml

Fixes : https://github.com/build-trust/ockam/issues/3626

## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -45,3 +45,8 @@ incremental = false
 codegen-units = 1
 inherits = "test"
 
+[package.metadata.cross.target.aarch64-unknown-linux-musl]
```
Comment (metaclips) [inline Cargo.toml]: Lets change `package.metadata.cross.target.aarch64-unknown-linux-musl` to `ockam_command.metadata.cross.target.aarch64-unknown-linux-musl`

File: Cargo.toml [hunk 2]
```diff
@@ -45,3 +45,8 @@ incremental = false
...
+dockerfile = "./tools/cross/Cross.Dockerfile.aarch64"
+
+[package.metadata.cross.target.armv7-unknown-linux-musleabihf]
+dockerfile = "./tools/cross/Cross.Dockerfile.armv7"
```
Comment (metaclips) [inline Cargo.toml:52]: `package.metadata.cross.target.armv7-unknown-linux-musleabihf` should be `ockam_command.metadata.cross.target.armv7-unknown-linux-musleabihf`

**Review by metaclips — CHANGES_REQUESTED** [review]
Thanks for an amazing pull request 🚀

**Comment by metaclips-botter** [comment]
❌ Orchestrator test on PR #3665 at commit `ed2c1acbeaf39092a313ffd31d64b45e56ca011e` failed

**Comment by mrinalwadhwa** [comment]
@duttaANI thank you for removing the merge commit in #3714 
Closing this and merging #3714

**Pull request closed without merging**